apix-integrationtests/src/main/java/eu/xenit/apix/tests/search/TermHitHighlightingTest.java

```diff
@@ -58,7 +63,10 @@ public void createHighlightTestTxt() throws InterruptedException {
             nodeService.setContent(target, inputStream, txtFile.getName());
             return null;
         }, false, true);
-        solrHelper.waitForSolrSync();
+        solrHelper.waitForTransactionSync();
+        solrHelper.waitForContentSync();
+        // Call bean representing the Transformers subsystem and start it. Subsystem is required for highlights.
+        transformersSubsystem.start();
```

Comment:
I thought we would not start the transformers subsystem manually, but just wait for solr to have indexed the text content of documents?

Reply:
must have slipped through in previous pass. its gone now